eng/common/scripts/git-branch-push.ps1

```diff
@@ -121,8 +121,11 @@ do
                 continue
             }
 
-            Write-Host "git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" commit -am `"$($CommitMsg)`""
-            git -c user.name="azure-sdk" -c user.email="azuresdk@microsoft.com" commit -am "$($CommitMsg)"
+            Write-Host "git add -A"
```

Comment:
I'm not sure how l likely it is for git add to fail but for consistency and completeness could you add the lastexitcode check after the call to git add here as well?

Reply:
Is this common one we'd be continuing from or something that would stop everything altogether? Should it have the standard `if ($LASTEXITCODE -ne 0)` write/continue that the other commands have?

Reply:
Yep. Good call.

Reply:
@JimSuplizio your comment didn't render when I originally responded to this thread. What'll happen with the change I added is that it'll leave the current loop, see that lastexitcode is still > 0, and never enter the try/finally again. Once it leaves the loop, it'll hit the very next `if LASTEXITCODE` on line 150 and exit with error.

Reply:
Ah, gotcha! Thanks for the info @scbedd